src/data/services/DiscoveryDataApiService.js

```diff
@@ -73,7 +73,18 @@ class DiscoveryDataApiService {
     // Create a promises array to handle all of the new/modified course runs
     const promises = courseRunData.map((courseRun) => {
       const courseRunUrl = `${DiscoveryDataApiService.discoveryBaseUrl}/course_runs/${courseRun.key}/`;
-      return apiClient.patch(courseRunUrl, courseRun);
+      // Minimal object needed to be written to Course Runs endpoint
+      const courseRunPatchBody = {
```

Comment:
I feel like there has to be a better way to send only the fields changed to discovery